mne/preprocessing/ica.py

```diff
@@ -270,6 +271,18 @@ def __repr__(self):
 
         return '<ICA  |  %s>' % s
 
+    def _check_for_unsupported_ica_channels(self, picks, info):
+        """Check for ica channels that are not considered
+        actual data channels. These prevents the program from
+        crashing without feedback when a bad channel is provided
+        to ICA whitening
```

Comment:
be more specific. Mention that it allows EOG channels as well as data channels in ICA.